scripts/components/FlProductItem.ts

```diff
@@ -29,6 +40,9 @@ export default class FlProductItem extends FlProductItemDesign {
             }
         });
     }
+    set itemTitleWidth(value: number) {
```

Comment:
is this maxWidth? if so name it correctly please. itemTitleMaxWidth and make the local variable same as function.

this.__itemTitleMaxWidth

Reply:
DONE